.gitignore

```diff
@@ -0,0 +1 @@
+*.sw?
```

Comment:
FYI if you add this to your global .gitignore you'll never have to add it to a project again